security/restricting-fiddler-features.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Restricting Fiddler
+description: "Using the Enterprise policy feature to disable specific Fiddler Everywhere features like remote connections, changing ports, cloud services, and others."
+slug: fe-restrict-policies
+publish: true
+position: 40
+---
+
+# Restricting Fiddler
+
+Users of Fiddler Everywhere often work in a secure corporate environment where specific configurations are imposed. That means that administrators need to have options to limit or hardcode particular features of the Fiddler Everywhere application. With the Fiddler Everywhere Enterprise tier, administrators can control a set of functionalities through device management solutions/software on macOS and Windows.
+
+## Restricting Features on macOS
+
+Users on macOS with administrative privileges can set the following policies through the preferred administrative tooling (like Jamf, Intune, etc.)
```

Comment:
We're suggestion a change to the sentences for a couple of reasons: 
- stated like it is right now it seems that Fiddler users who have admin rights could apply policies to themselves 
- we'd like to use "app configuration" instead of "policies". 

IT teams managing macOS systems can apply app configuration through the preferred administrative tooling (like Jamf, Intune, etc.) using the following keys: 

Let me know what you think.